src/signature.rs

```diff
@@ -307,44 +306,40 @@ impl Recipient {
                         .truncate(true)
                         .open(recipient_sig_filename)?;
 
-                    recipient_sig_file.write(&output)?;
+                    recipient_sig_file.write_all(&output)?;
                 }
             }
         }
-
-        //TODO: readd this
-        //PasswordEntry::reencrypt_all_password_entries(store)?;
-
```

Comment:
@cortex this was maybe a strange delete, looks important?

Reply:
ahh, yes,  that is not a clippy thing, but I did actually already add it back, so the TODO is done:
https://github.com/cortex/ripasso/blob/master/src/pass.rs#L373
https://github.com/cortex/ripasso/blob/master/src/pass.rs#L382